ci-openvdb/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM aswfstaging/ci-base:2019
+
+COPY scripts/* /tmp/
+
+RUN /tmp/install_openexr.sh 2.3.0 && \
+    /tmp/install_log4cplus.sh 1.1.2 && \
+    /tmp/install_glfw.sh 3.0 && \
```

Comment:
Could we try bumping this to 3.0.4 please? I see there have been some build environment improvements between 3.0.0 and 3.0.4. If that doesn't work, we'll probably need some CMake updates on our side.

Reply:
Actually, we just discussed in our TSC meeting and decided that we'd move to a minimum of 3.1.2 for GLFW if you don't mind bumping that value again @aloysbaillet? I looped you in on the email thread for context.

Reply:
Building away... fingers crossed!

Reply:
Of course the build failed as new glfw needs libXinerama :-)
Another build on its way there: https://dev.azure.com/aloysbaillet/ASWF/_build/results?buildId=143&view=results
No associated commit on this branch as I can't seem to push to this repo using https from work...